spec/factories/people.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :person do
+    first_name { 'Herbert' }
+    last_name { 'Herbertson' }
+    phone_number { '+4990909090' }
+    email { "herbert.herbertson@hpi.de" }
```

Comment:
users didn't allow multiple users with the same mail, but people do now. Expected behavior or missing validation?

Reply:
Expected behavior. Multiple admins may want to register using their personal email addresses, but only provide admin@hpi.de on their public person.